config-service-api/src/main/proto/org/hypertrace/config/service/v1/config_service.proto

```diff
@@ -78,6 +78,12 @@ message ContextSpecificConfig {
 
   // config value which applies to the above context
   google.protobuf.Value config = 2;
+
+  // timestamp (in milliseconds since epoch) at which this config was created
+  int64 creation_timestamp = 3;
+
+  // timestamp (in milliseconds since epoch) at which this config was last updated
+  int64 update_timestamp = 4;
```

Comment:
Haven't added these fields to `getConfig` API because that API can merge the configs belonging to different contexts, each of which will have different creation and update timestamps.

Reply:
That makes sense. Eventually, we may have to expose an API that doesn't merge and does expose those (wish getConfig had been named getMergedConfig or something...), but no need until we have a use case. The same argument could actually be made for this API since the main use case as I understand it is internal - the sorted results, but I'm fine either way since it's done.